### DISCUSSION TITLE
1.9.0-alpha

### DISCUSSION THREAD

**Discussion by SkalskiP** [SkalskiP/make-sense#266] [Announcements]
# [1.9.0-alpha] - 2022-07-26

## 🚀 Added

- Turning annotation visibility on and off (https://github.com/skalskip/make-sense/issues/255)
- Introduction and configuration of the linter (https://github.com/SkalskiP/make-sense/pull/262)
- Adding headers to CSV files created as a result of exporting box annotations (https://github.com/skalskip/make-sense/issues/252) 

https://user-images.githubusercontent.com/26109316/181107825-6f5fb7bc-9cf4-47a1-bf52-c3881f853cab.MP4

## 🌱 Changed

- Migrate to the latest versions of React and MaterialUI (https://github.com/SkalskiP/make-sense/pull/262)
- Migrate from Webpack to Vite (https://github.com/SkalskiP/make-sense/pull/262)

## 🔧 Fixed
- Fix docker installation problems (https://github.com/SkalskiP/make-sense/issues/210)
- Fix PASCAL XML export to work with TFLite Model Maker (https://github.com/SkalskiP/make-sense/pull/214)

## 🏆 Contributors

Special thanks are due to @rasyidf for his exceptional activity, both in the form of PRs, as well as great ideas for feature development. Than you! 🙏

 @rasyidf, @IuliuNovac, @Parthiban

## 🚧 Work in progress

- One of the definite highlights of the next release will be the introduction of the ability to load any custom YOLOv5 model into MS, and use it when labeling images. Thanks to @IuliuNovac's tremendous work, we already have an advanced and working proof of concept for this feature (https://github.com/SkalskiP/make-sense/pull/220).

<hr /><em>This discussion was created from the release <a href='https://github.com/SkalskiP/make-sense/releases/tag/'>1.9.0-alpha</a>.</em>

**Comment by rasyidf** [comment]
I'm very excited about this, 🚀🚀

**Comment by SkalskiP** [comment]
Me too! I have already started working on the next features.
